ts/screens/authentication/InterruptedReadingCardScreen.tsx

```diff
@@ -0,0 +1,76 @@
+import { Container, H2, Text, View } from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import BaseScreenComponent from "../../components/screens/BaseScreenComponent";
+import FooterWithButtons from "../../components/ui/FooterWithButtons";
+import I18n from "../../i18n";
+import variables from "../../theme/variables";
+
+type OwnProps = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+
+type Props = OwnProps;
+
+const styles = StyleSheet.create({
+  contentContainerStyle: {
+    padding: variables.contentPadding
+  },
+  text: {
+    fontSize: variables.fontSizeBase
+  }
+});
+
+class InterruptedReadingCardScreen extends React.Component<Props> {
+  constructor(props: Props) {
```

Comment:
```suggestion
  constructor(props: Props) {
```
since you are not doing other operations you can remove the constructor

Reply:
Ok